### PR TITLE
[WIP] UX

### PR DESCRIPTION
**Pull request by grena**

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+  }
+}
+
+.submit-form {
+  .btn.btn-primary.dropdown-toggle {
+    background-color: darken(@color-new-green, 10%);
+    width: 34px;
+
+    .caret {
+      margin-left: 0px;
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: Recess warning : "No need to specify units when a value is 0"

Comment (grena) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: Yep, fixed, thank you!

File: src/Pim/Bundle/DataGridBundle/Resources/public/css/grid.less [hunk 2]
```diff
@@ -10,7 +10,7 @@ table.grid {
     border-left: 4px solid transparent;
   }
   thead tr th.ascending span.caret {
-    margin-top:4px;
+    margin-top:10px;
```
Comment (skeleton) [inline src/Pim/Bundle/DataGridBundle/Resources/public/css/grid.less]: add space before 10px please

File: src/Pim/Bundle/EnrichBundle/Resources/views/AssociationType/edit.html.twig [hunk 3]
```diff
@@ -28,7 +28,7 @@
                 confirmationMessage,
                 'flash.association type.removed'|trans
             ) }}
-            {{ elements.submitBtn() }}
+            {{ elements.submitBtn('', 'ok') }}
```
Comment (skeleton) [inline src/Pim/Bundle/EnrichBundle/Resources/views/AssociationType/edit.html.twig]: there is no translation key ?

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+    width: 34px;
+
+    .caret {
+      margin-left: 0;
+    }
+  }
+}
+
+
+
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove extra blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+    .icon-button();
+    background-color: darken(#a1a9b7, 10%);
+    border-top-left-radius: 2px;
+    border-bottom-left-radius: 2px;
+  }
+
+  &:hover {
+    background-color: darken(#a1a9b7, 10%);
+  }
+
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove extra blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 6]
```diff
@@ -0,0 +1,366 @@
...
+    line-height: 28px;
+    margin: 8px 5px;
+    border-radius: 2px;
+    border: none;
+    background-color: #fff;
+
+    .caret {
+      margin-top: 12px;
+    }
+  }
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: indentation

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 7]
```diff
@@ -7,6 +7,12 @@ body {
   color: @textColor;
   background-color: @backgroundColor-white;
 
+  &.login-page {
+    background-image: url("/bundles/pimui/images/background.svg");
+    background-size: cover;
+    background-color: #f1f4f9;
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: you can call @color-new-blue-light

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 8]
```diff
@@ -669,7 +680,7 @@ h3.tab-header {
 
 .attribute-actions {
   .attribute-edit-actions {
-    margin-right: 10px;
+    margin-right: 0px;
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: "px" is useless

File: src/Pim/Bundle/UIBundle/Resources/public/css/tabnav.less [hunk 9]
```diff
@@ -0,0 +1,68 @@
+@import 'less/variables';
+@import 'less/mixins';
+
+.navigation.navbar-extra {
+  padding: 15px;
+  background-color: #F7F7F9;
+
+  .sub-title{
+    color: #9452BA;
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tabnav.less]: use @color-new-purple-medium

File: src/Pim/Bundle/UIBundle/Resources/public/css/accordion.less [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+.accordion {
+  margin-bottom: 0;
+
+  .accordion-group {
+    margin: 0;
+    border-radius: 0;
+    border-top: none;
+
+    .accordion-heading {
+      background-color: #E7E7EC;
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/accordion.less]: use @color-new-grey

File: src/Pim/Bundle/UIBundle/Resources/public/css/form.less [hunk 11]
```diff
@@ -785,17 +785,33 @@ div.dropdown-menu.channel {
...
+      top: 0;
+      right: 0;
+      width: 34px;
+      height: 34px;
+      line-height: 34px;
+      margin: 0;
+      background-color: #CCC;
+      text-align: center;
+      line-height: 34px;
+      background-color: darken(#E7E7EC, 10%);
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: use @color-new-grey

File: src/Pim/Bundle/UIBundle/Resources/public/css/tabnav.less [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+      }
+    }
+  }
+}
+
+.tab-pane {
+  padding-top: 15px;
+
+  &.buffer-top { padding-top: 0}
+
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tabnav.less]: remove extra blank line

File: src/Pim/Bundle/UIBundle/Resources/public/css/tabnav.less [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+        border: solid 1px @color-new-purple-medium;
+      }
+    }
+  }
+}
+
+.tab-pane {
+  padding-top: 15px;
+
+  &.buffer-top { padding-top: 0}
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/tabnav.less]: put `padding-top: 0` in a new line

**Pull request merged**